src/test/java/core/basesyntax/ShopOperationsDaoCsvImplTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax;
+
+import dao.ShopOperationsDao;
+import dao.ShopOperationsDaoCsvImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ShopOperationsDaoCsvImplTest {
+    private static final String inputFileName = "fruit-shop.csv";
```

Comment:
```suggestion
    private static final String INPUT_FILE = "fruit-shop.csv";
```
String - deeply immutable 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Так, смотри, то, что immutable, то капсом, типа Стринга и наших immutable классов. Всё остальное не immutable - не капсом, а как обычно.
Например,
private static final String str;
private static final NonImmutableCat cat;
private static final ImmutableCat CAT;
Верно?